packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added an `encoding` option to the `zowe files search data-sets` command to enable users to search data sets using another codepage. [#2161](https://github.com/zowe/zowe-cli/issues/2161)
```

Comment:
Would like to make this more action driven, please: Issue the `aaa` command with the new `bbb` option to use a different codepage when reading data set content.

Reply:
Updated changelog entry.